Makefile.armv7m

```diff
@@ -49,29 +49,27 @@ else ifeq ($(TARGET_SUBFAMILY), stm32l4x6)
 	VADDR_KERNEL_INIT=8000000
 	KERNEL_TARGET_DEFINE=-DCPU_STM32L4X6 -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt105x)
-	RAM_SIZE=128
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT105X -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt106x)
-	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT106X -DNOMMU
 else ifeq ($(TARGET_SUBFAMILY), imxrt117x)
-	RAM_SIZE=256
 	VADDR_KERNEL_INIT=0
-	KERNEL_TARGET_DEFINE=-DCPU_IMXRT117X -DNOMMU 
+	KERNEL_TARGET_DEFINE=-DCPU_IMXRT117X -DNOMMU
 else
 	$(error Incorrect TARGET.)
 endif
 
 
-LDFLAGS = -z max-page-size=0x10 --gc-sections -q
+LDFLAGS = -z max-page-size=0x10 --gc-sections
 
 ifeq ($(KERNEL), 1)
 	CFLAGS += -DRAM_SIZE=$(RAM_SIZE) $(KERNEL_TARGET_DEFINE) -ffixed-r9
 	LDFLAGS += -Tbss=20000000 -Tdata=20000000
 else
 	CFLAGS += -fpic -fpie -msingle-pic-base -mno-pic-data-is-text-relative
+	LDFLAGS += -q
```

Comment:
I would gladly welcome comment what this does, but I know it's a political decision

Reply:
Do you think about describing purpose of each of the flag? I can do it.
However like you wrote: it is political decision so that I would like to know the opinion of @pawelpisarczyk and @agkaminski.

Reply:
IMHO it is very welcome to add **concise** comment on non-trivial pieces of code/Makefile/anything

Reply:
so maybe we can add some comment? maybe `output .rel.* sections to make ELF position-independent`?

Reply:
Done